find-missing-results.js

```diff
@@ -45,12 +45,10 @@ const generateReportMap = (allResults) => {
     if (!allResults) {
       if (browserKey == 'ie') {
         // Ignore super old IE releases
-        result[browserKey] = result[browserKey].filter((v) => v >= '6');
+        result[browserKey] = result[browserKey].filter((v) => v >= 6);
```

Comment:
Can you use `compareVersions` here so that the string&rarr;number conversion isn't implicit?